Tests/SonarQube.TeamBuild.PreProcessor.Tests/PreProcessorTests.cs

```diff
@@ -122,35 +85,6 @@ public void PreProc_FileProperties_Supplied()
         }
 
         [TestMethod]
-        public void PreProc_NoHostUrl()
-        {
```

Comment:
No longer necessary. The argument processor is responsible for validating the URL exists and the pre-processor class won't be called unless the arguments are valid.
The new unit test above that shells the PreProcessor executable checks this is the case.